### ISSUE TITLE
CTrl + F ?

### ISSUE THREAD

**Issue by kikecalpe** [Guake/guake#34]
CTrl + F ?  http://guake.org/ticket/529
Reported by:    fredm   Owned by:   somebody
Priority:   major   Milestone:  0.5.0
Component:  guake   Version:  
Keywords:       Cc:  
Description

It seems to have no possible to do a search in the terminal ?

Could you add it ?

In parallel, it s possible to save all the shell opened ? When I reboot, could I rehave the shells of my previous session ?

Regards, Frédéric

**Comment by frederickjh** [comment]
Duplicate of #116
